src/cmd/deploy_nsfs.sh

```diff
@@ -0,0 +1,419 @@
+#!/bin/bash
+
+### REQUIREMENTS ###
+# 1. NooBaa system installed on minikube
+# 2. kubectl cli installed
+# 3. aws cli installed
+# 4. noobaa cli installed
+
+# the results of this installation are - 
+# 1. noobaa nsfs exported bucket (RPC) - export of pre existing file system directory
+# 2. noobaa nsfs bucket (S3) - noobaa will create the file system directory during the request
+
+
+HELP="
+Help:
+
+    "deploy_nsfs" is a noobaa-core command deploys different resources in order to use namespace FS feature.
+    S3 bucket - is a NooBaa NSFS bucket that its underlying directory doesn't exist before the bucket creation request, NooBaa will create it during the bucket creation process.
+    Exported bucket (RPC) - is a NooBaa NSFS bucketthat its underlying directory exist before the bucket creation request, NooBaa will check the directory's existance during the bucket creation process.
+    For more information refer to the noobaa docs.
+";
+
+USAGE="
+Usage:
+    noobaa-core deploy_nsfs <deployment> <cleanup> [options...]
```

Comment:
`Usage: noobaa-core deploy_nsfs  [options]
Options:
-h, --help    : Print this help message
deployment : To deploy NSFS on K8S
cleanup : To cleanup the deployment of NSFS on K8S